app/src/androidTest/java/org/mozilla/focus/utils/AndroidTestUtils.java

```diff
@@ -32,10 +32,10 @@
 public final class AndroidTestUtils {
 
     public static void beforeTest() {
-        beforeTest(true);
+        beforeTest(true, true);
     }
 
-    public static void beforeTest(final boolean firstRun) {
+    public static void beforeTest(final boolean disableOnBoarding, final boolean disableRateAppCheck) {
```

Comment:
These two parameters get confusing now. Could you please make a builder for this? thx!